mlflow/pipelines/steps/train.py

```diff
@@ -158,6 +172,104 @@ def _run(self, output_directory):
             raw_validation_df, prediction_result, self.target_col
         )
 
+        from mlflow.entities import ViewType
```

Comment:
Can we pull this section out as a separate function? It's getting really long.

Reply:
Done